tutor/src/screens/task/ux.js

```diff
@@ -104,11 +104,16 @@ export default class TaskUX {
     step.answer_id = answer.id;
     step.is_completed = true;
     await step.save();
-    // don't advance if there's feedback so the user can view it
-    if (!this.task.isFeedbackAvailable) {
-      this.onAnswerContinue(step);
-    } else {
-      this.moveToStep(step);
+
+    if (
+      step.multiPartGroup &&
+      this.task.isFeedbackAvailable &&
+      this._stepIndex > this.steps.indexOf(step)
+    ) {
```

Comment:
should this be `this._stepIndex != this.steps.indexOf(step)` ?  If we only check for greater then it won't scroll up if you select a previous step